libraries/tensorflow-gpu-2.0/Dockerfile

```diff
@@ -0,0 +1,2 @@
+RUN pip install keras==2.1.4
```

Comment:
@zeryx do you remember why we added `keras` to the dockerfile?

Reply:
yeah, version control - keras wildcards which version of tensorflow to install which leads to incompatibility.